packages/application-shell/src/components/app-bar/app-bar.js

```diff
@@ -12,14 +17,18 @@ import messages from './messages';
 import styles from './app-bar.mod.css';
 
 export const BackToProjectLink = props => (
-  <Link to={`/${props.projectKey}`}>
-    <Spacings.Inline alignItems="center" data-test="back-to-project">
-      <AngleLeftIcon theme="green" size="small" />
-      <Text.Detail intlMessage={messages.backToProjectLink} />
-    </Spacings.Inline>
-  </Link>
+  <LinkButton
+    to={`/${props.projectKey}`}
+    iconLeft={<AngleLeftIcon />}
+    label={
+      <FormattedMessage {...messages.backToProjectLink}>
+        {txt => txt}
```

Comment:
🤔

Reply:
I don't want to injectIntl and this component needs a string.